packages/backend/migrations/20230202124709_create_ledger_transfers_table.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('ledgerTransfers', function (table) {
+    table.uuid('id').notNullable().primary()
+    table.uuid('transferRef').notNullable().unique().index()

```

Comment:
`transferRef` is the caller-assigned reference, while `id` is the internal table id